config

```diff
@@ -193,12 +193,22 @@ if [ $ngx_found = yes ]; then
     $ps_src/ngx_server_context.cc \
     $ps_src/ngx_url_async_fetcher.cc"
 
-  # Make pagespeed run immediately before gzip.
+  # Make pagespeed run immediately before gzip and Brotli.
+  if echo $HTTP_FILTER_MODULES | grep ngx_http_brotli_filter_module >/dev/null; then
+    module=ngx_http_brotli_filter_module
+  elif [ $HTTP_GZIP = YES ]; then
+    module=$HTTP_GZIP_FILTER_MODULE
+  else
+    module=$HTTP_RANGE_HEADER_FILTER_MODULE
```

Comment:
Is HTTP_RANGE_HEADER_FILTER_MODULE guaranteed to be there?

(Regardless, your change is an improvement, since previously if there was no `HTTP_GZIP_FILTER_MODULE` it looks like we wouldn't install ourselves.